docs/dev.rst

```diff
@@ -64,39 +68,45 @@ Setup Quickstart
 
    From the root of this repository, run::
 
-     $ make build
+      $ make build
 
    That will build the app Docker image required for development.
 
 5. Initialize Postgres, Elasticsearch, S3, and Pub/Sub.
 
-   Then you need to set up services. To do that, run::
+   Then you need to set up services. To do that, run:
+
+   .. code-block:: shell
 
-     $ make runservices
+      $ make runservices
 
-   This starts service containers. Then run::
+   This starts service containers. Then run:
 
-     $ make setup
+   .. code-block:: shell
+
+      $ make setup
 
    This creates the Postgres database and sets up tables, stored procedures,
    integrity rules, types, and a bunch of other things. It also adds a bunch of
    static data to lookup tables.
 
-   For Elasticsearch, it sets up Supersearch fields and the index for
+   For Elasticsearch, it sets up Super Search fields and the index for
    processed crash data.
 
    For S3, this creates the required buckets.
 
-   For Pub/Sub, this creates topics and subscriptions.
+   For SQS, this creates queues.
```

Comment:
We're currently using SQS and not Pub/Sub. I should have updated this a couple of years back.